Parsers/ASimDns/ProductParsers/ASimDnsMicrosoftSysmon.yaml

```diff
@@ -15,88 +15,88 @@ References:
   Description: ASIM Sysmon DNS Parser (event number 22) from "Event" and "WindowsEvent" tables.
 ParserName: ASimDnsMicrosoftSysmon
```

Comment:
I think that I did not see a version update for this one.

Reply:
this is the first version for this parser